server/src/main.rs

```diff
@@ -134,32 +135,38 @@ fn main() {
 
         WipeMode::wipe(
             cli_args.wipe.as_ref(),
-            config.meta.storage_path().into(),
-            config.worker.storage_path().into()
+            config.node.meta.storage_path().into(),
+            config.node.worker.storage_path().into()
         ).await.expect("Error when trying to wipe the configured storage path");
 
-        let app = Application::new(
-            config.node_ctrl,
-            config.meta,
-            config.worker,
-            config.admin,
-            config.bifrost,
-        );
+        let controller_location = if config.roles.contains(Role::Controller) {
+            ControllerLocation::SpawnLocal(config.node.clone())
+        } else {
+            ControllerLocation::Remote(config.remote_controller.expect("Either 'controller_endpoint' or controller role needs to be configured for server"))
```

Comment:
error is not consistent with the config name perhaps?

Reply:
Good catch. This is wrong. Will update it.